examples/README.md

```diff
@@ -0,0 +1,48 @@
+# Merlin Models Example Notebooks
+
+We have created a collection of Jupyter notebooks based on different datasets. These example notebooks demonstrate how to use Merlin Models with TensorFlow.
+
+## Available Example Notebooks
+
+1. **[Getting Started](01-getting-started.ipynb)**: It provides a getting started with Merlin Models. We will train [Facebook's DLRM](https://arxiv.org/pdf/1906.00091.pdf) architecture with only 3 commands.
```

Comment:
I'd repeat the title of the notebook so that when customers follow the link, they can be assured that they found the right content.  If the titles are different, it's slightly jarring.

"1. **[Getting Started with Merlin Models: Develop a Model for MovieLens](...)**: Learn the basics of Merlin Models and train a model with [Facebook's DLRM](https://...) architecture with only 3 lines of code."

Reply:
Oh, geez.  You are correct Benedikt, I'm sorry for not noticing.  I agree with your instinct that it'll be better to change the file name, even if it's `01-getting-started-model-movielens.ipynb`.